src/sharing/index.jsx

```diff
@@ -263,6 +264,7 @@ const OwnerSharingModal = ({ document, ...rest }) => (
             createContact={createContact}
             recipients={getRecipients(document.id)}
             link={getSharingLink(document)}
+            isShared={byDocId[document.id] !== undefined}
```

Comment:
Your commit message is `changed file extension`
But in the commit I see three changes
- the file has been renamed
- the function parameter has been changed
- a props in a third component has been added

Something is wrong with the commit:
1. if every changes are tighted together, then change the commit message (use the infinite form with a capital letter)
1. if changes are decorrelate (what I think), split your changes into several commits with good and informative commit messages

That is important because there is a lot of people working on the code and the git history (`git log`) is very useful to follow the code changes and understand the past, the present and be prepared for the future.